core/test/unit/adapters/scheduling/SchedulingDefault_spec.js

```diff
@@ -60,6 +60,55 @@ describe('Scheduling Default Adapter', function () {
             ]);
         });
 
+        it('reschedule: default', function (done) {
+            sinon.stub(scope.adapter, '_pingUrl');
+
+            const time = moment().add(200, 'milliseconds').valueOf();
+
+            scope.adapter.schedule({
+                time: time,
+                url: 'something',
+                extra: {
+                    oldTime: null,
+                    method: 'PUT'
+                }
+            });
+
+            scope.adapter.reschedule({
+                time: time,
+                url: 'something',
+                extra: {
+                    oldTime: time,
+                    method: 'PUT'
+                }
+            });
+
+            setTimeout(() => {
+                scope.adapter._pingUrl.calledOnce.should.eql(true);
+                done();
+            }, 250);
```

Comment:
We probably can reduce the timeouts to achieve a quicker execution of the test

Reply:
reduced timeouts to 20 and 50 ms, as that seemed to be enough